RestSharp/Authenticators/OAuth/Extensions/OAuthExtensions.cs

```diff
@@ -32,12 +39,29 @@ public static OAuthSignatureMethod FromRequestValue(this string signatureMethod)
             }
         }
 
-        public static string HashWith(this string input, HashAlgorithm algorithm)
+
+        public static string HashWithHMACSHA1(this string input, string key)
```

Comment:
Whole `System.Security.Cryptography`  namespace is not available in UWP. The whole hashing would be better to refactored by adding a separate class handling that and class would be different for UWP and other targets. But  I decided to do as little changes as possible and refactor it later if needed.